bofire/domain/features.py

```diff
@@ -170,7 +214,8 @@ def validate_experimental(self, values: pd.Series, strict=False) -> pd.Series:
 
         Args:
             values (pd.Series): A dataFrame with experiments
-            strict (bool, optional): Boolean to distinguish if the occurence of fixed features in the dataset should be considered or not. Defaults to False.
```

Comment:
I would prefer using type hints over of types in the documentation

Reply:
Hmm, I do not understand what you mean ...

Reply:
There is no type hint for the variable fixed, but there is a comment saying it should be bool and that it is optional. Btw., here, optional means that it has a default parameter. But if you use `Optional[bool]` it means it can be `None` independent of the existence of a default parameter.
![image](https://user-images.githubusercontent.com/50267830/202464553-f7307b60-ad45-4000-86bd-e7fe930564b7.png)